Documentation/Dev/Database Design - Milestone 1.md

```diff
@@ -0,0 +1,38 @@
+# Milestone #1 - Database Design
+This is a summary of the different entities and relationships extracted from our functional requirements for the first milestone.
+
+## THIS IS A WiP Documentation, please refer to the BDSetup.js file for a more up-to-date version. This is currently OUT OF DATE
+
+## Table & Column Naming Conventions:
+- Tables always start with a capital, have no spaces, and are camel cased. eg: "SearchableInfo" (Upper Camel Case aka Pascal Case) 
+- Tables that are intended to store additional information to be referenced by an ID in another table have the postfix "Type". eg: "GenderType"
+- Columns start with a lower case letter and are camel cased eg: "birthYear" (lower Camel Case aka Dromedary Case)
+
+## Member
+### memberID - Auto-Incrementing Primary Key
+Type: INT
+
```

Comment:
This all needs to be updated, I might remove it from here and add it to gitignore and keep it local until it's up to date so nobody gets confused.